### PR TITLE
Spectra v2.7.5

### PR DESCRIPTION
**Pull request by adi3890**
* Improvement: Container - Added an opacity setting for the overlay of the image and video background types.
* Improvement: Info Box - Added heading tag settings for the prefix.
* Improvement: Info Box - Added an inherit-from-theme option for the text-type CTA.
* Improvement: Form - Added an inherit-from-theme option for the submit button styling.
* Improvement: Global - Enhanced the reusable block functionality for the following blocks: Review, Content Timeline, Post Timeline, Price List, Separator, Social Share, Star Rating, Table Of Contents, Taxonomy List, Team, Testimonials.
* Fix: Call To Action - Resolved an issue in which the bottom margins of the title and description would not work as intended for Tablet and Mobile devices on the frontend.
* Fix: Blockquote - Resolved an issue wherein having '#' or '@' symbols in the description would result in the removal of content following the symbols when shared on Twitter.
* Fix: Post Carousel - Resolved an issue wherein the carousel dots would not appear centered.
* Fix: Social Share - Resolved an issue wherein the gap between icons would add unwanted margins to the sides of the wrapper.
* Fix: Testimonials - Resolved an issue wherein the Testimonial dots would not appear centered.
* Fix: Global - Resolved an issue that was encountered when updating to WordPress 6.3 on some websites.
* Fix: Global - Resolved an issue wherein the color-picker default icon would not appear in WordPress 6.3.
* Fix: Admin - Resolved an issue in which an undefined record could appear on the Instagram integrations page.

### CODE REVIEW

**Review by rahulvarma722 — APPROVED** [review]

**Review by rahulvarma722 — APPROVED** [review]

File: src/blocks/info-box/components/CTA.js [hunk 1]
```diff
@@ -18,82 +18,109 @@ const InfoBoxCta = ( props ) => {
...
+		{ attributes.ctaIconPosition === 'after' && ctaIconOutput }
+		</>
+	);
 
 	if ( setAttributes !== 'not_set' ) {
 		return (
 			<div className="uagb-ifb-cta">
-				{ attributes.ctaType === 'text' && (
+				{ ( attributes.ctaType === 'text' && attributes.inheritFromTheme ) && (
+					<div className={ classnames( 'uagb-ifb-button-wrapper', 'uagb-ifb-button-type-text', 'wp-block-button' ) }>
```
Comment (bsf-zanev) [inline src/blocks/info-box/components/CTA.js:43]: Can we replace `classnames` with `uagbClassNames` in this file?

File: src/blocks/testimonial/edit.js [hunk 2]
```diff
@@ -102,9 +103,9 @@ const UAGBtestimonial = ( props ) => {
 
 	useEffect( () => {
 		if ( equalHeight ) {
-			uagb_carousel_height( clientId.substr( 0, 8 ) );
+			uagb_carousel_height( block_id );
 		} else {
-			uagb_carousel_unset_height( clientId.substr( 0, 8 ) ); // eslint-disable-line no-undef
+			uagb_carousel_unset_height( block_id ); // eslint-disable-line no-undef
```
Comment (bsf-zanev) [inline src/blocks/testimonial/edit.js:108]: Can we remove the `no-undef` comment here by adding this to our eslintrc globals?

**Review by rajmohanreddy774 — APPROVED** [review]

File: src/blocks/info-box/components/CTA.js (hunk 1 above)
Comment (adi3890) [reply src/blocks/info-box/components/CTA.js:43]: nothing major lets update it after release.

File: src/blocks/testimonial/edit.js (hunk 2 above)
Comment (adi3890) [reply src/blocks/testimonial/edit.js:108]: lets update it after release.

**Pull request merged**